webapp/src/App.js

```diff
@@ -187,7 +187,7 @@ function AppRoutes() {
           )}
         />
         <Route
-          path="/offering-product/:id"
+          path="/offerings/:offeringId/products/:productId"
```

Comment:
This URL is too long/fragmented for a client, too much can go wrong. Can we do something like `/product/:offeringId-:productId`?

Reply:
This worked.